steps/cloud/azure/upload-storage-account.yml

```
@@ -28,7 +28,7 @@ steps:
     fi
   displayName: "Upload Result"
   env:
-    SOURCE_FILE_NAME: ${{ parameters.source_file_name }}
+    SOURCE_FILE_NAME: $(TEST_RESULTS_FILE)
```

```suggestion
    SOURCE_FILE_NAME: ${{ parameters.source_file_name }}
```
can we revert this change, so that we allow user to override the file name ?

---

If we don't want user to override, we might have to remove parameter and use the env variable `TEST_RESULTS_FILE` in all the places

---

reverted, nice catch!